app/controllers/boards_controller.rb

```diff
@@ -22,13 +22,25 @@ def edit
 
   def update
     @board = Board.find(params[:id])
```

Comment:
Ta linijka ( szukanie board'a ) powtarza sie w wielu miejsach. Przenies to do metody.